packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualLiquidatable.sol

```diff
@@ -482,7 +482,6 @@ contract PerpetualLiquidatable is PerpetualPositionManager {
             withdrawalAmount = collateral.add(finalFee);
             delete liquidations[sponsor][liquidationId];
         }
-        require(withdrawalAmount.isGreaterThan(0), "Invalid withdrawal amount");
```

Comment:
Removed to save bytecode. Open to opinions, downside is that a liquidator/disputer/sponsor could spam the contract by calling withdraw liquidation even if they have no rewards.